engine/table/src/main/java/io/deephaven/engine/table/impl/select/BaseIncrementalReleaseFilter.java

```diff
@@ -212,6 +212,14 @@ public void setRecomputeListener(RecomputeListener listener) {
     @Override
     abstract public BaseIncrementalReleaseFilter copy();
 
+    protected void onCopy(BaseIncrementalReleaseFilter copy) {
+        copy.started = started;
+        copy.initialized = initialized;
+        if (started && initialized) {
+            copy.addToUpdateGraphProcessor();
+        }
```

Comment:
I don't think we should copy this state. "started" seems instance-specific.

Reply:
As discussed offline I changed this to throw an UnsupportedOperation.